LaTeX_guidelines.md

```diff
@@ -0,0 +1,66 @@
+# LaTeX Guidelines for The Final Report
+Version on Dec 24, 2020.
+
+Author: Ezgi.
+
+Assuming that anyone has some sort of familiarity with LaTeX, these are meant to be just *friendly reminders* about how to write the final report.
+
+1. In the report, we need to label *everything* that we refer to: figures, equations, sections, subsections, subsubsections etc. 
+
+For figures, do:
+```tex
+\label{fig:XX}
+```
+For equations, do:
+```tex
+\label{eq:XX}
+```
+For sections, do:
+```tex
+\label{sec:XX}
+```
+For subsections, do:
+```tex
+\label{subsec:XX}
+```
+I hope you get the idea :)
+
+To refer to these labels, let's say a figure, in the text simply do:
+```tex
+\ref{fig:XX}
+```
```

Comment:
Would add a note on doing `Figure~\ref{fig:XX}`

Reply:
why ~ ?

Reply:
https://www.andy-roberts.net/writing/latex/floats_figures_captions

Reply:
For my projects sometimes I have an extra macro for this (\reffig), but sometimes you also want Fig.~\ref, so I lost use of that and I can't remember where I last used it

Reply:
I doubt that people will read this document and put the '~' before the 'Figure'... but fingers crossed -- revising now